samples/precompiled/HttpSyncStart.cs

```diff
@@ -2,6 +2,7 @@
 using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
+using ImpromptuInterface.Dynamic;
```

Comment:
Why is ImpromptuInterface.Dynamic needed here?

Reply:
Great catch! This was left my testing for another approach. I will remove it shortly.